main.go

```diff
@@ -157,6 +158,11 @@ var debArchMap = map[string]string{
 	"arm64": "arm64",
 }
 
+var apkArchMap = map[string]string{
+	"amd64": "amd64",
+	"arm64": "arm64",
+}
```

Comment:
This does not look like it is being used. Apk and deb use the same arch naming scheme, so it is not really needed.